bitbox-bridge/release/windows/wix/signtool.cmd

```diff
@@ -1 +1 @@
-signtool.exe sign /d "BitBoxBridge" %*
+E:\shiftcrypto\CODESIGN\sign.cmd %*
```

Comment:
would it make sense to keep this file as is. well, add it (wis/signtool.cmd) to .gitignore and instead add instructions in the readme with an example contents? mine for example is not on `E:` but some diffent drive letter

Reply:
or maybe simply `sign.cmd %` and instruct to put the location of it into PATH?